src/blob_gc.h

```diff
@@ -46,8 +56,10 @@ class BlobGC {
   bool trigger_next() { return trigger_next_; }
 
  private:
-  std::vector<BlobFileMeta*> inputs_;
-  std::vector<BlobFileMeta*> sampled_inputs_;
+  std::vector<BlobFileMeta*> gc_inputs_;
+  std::vector<BlobFileMeta*> fs_inputs_;
+  std::vector<BlobFileMeta*> gc_sampled_inputs_;
```

Comment:
This variable is more confusing

Reply:
fixed

Reply:
But I did not see the modification of the variable name.